src/components/organisms/Room/Participant.scss

```
@@ -0,0 +1,59 @@
+@import "scss/constants";
+
+.participant {
```

Good opportunity for proper BEM, not just copy old obsolete practices

```suggestion
.Participant {
```

---

c4879fa